src/components/molecules/ContentSearchbar.unit.js

```diff
@@ -0,0 +1,71 @@
+import ContentSearchbar from "./ContentSearchbar";
+
+const contentSearchbarComponent = {
+	components: { ContentSearchbar },
+	data: () => ({
+		value: "",
+		placeholder: "Suche nach...",
+	}),
+	template: '<content-searchbar v-model="value" :placeholder="placeholder" />',
+};
+
+describe("@components/molecules/ContentSearchbar", () => {
+	it(...isValidComponent(ContentSearchbar));
+
+	it("Defaults render", async () => {
+		const wrapper = mount(contentSearchbarComponent);
+		expect(wrapper.find("input").exists()).toBe(true);
+		expect(wrapper.find(".search__container--icon").exists()).toBe(true);
+	});
+
+	it("Updates the v-model when typing", async () => {
+		const testString = "teststring";
+		const wrapper = mount(contentSearchbarComponent);
+		const inputField = wrapper.find("input");
+		expect(wrapper.vm.value).toBe("");
+		inputField.trigger("focus");
+		inputField.setValue(testString);
+		expect(wrapper.vm.value).toBe(testString);
+	});
+
+	it("Emits input event on ANY keypress in input", async () => {
```

Comment:
```suggestion
	it("Emits input event on any input value change", async () => {
```
I would suggest this name, because you are not testing any keypress events with this test.